languages/Clojure/deromanize

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+root="./languages/Clojure/lib/deromanize/target/uberjar"
```

Comment:
Does this mean that it has to be run from the root directory and can't be run from the `languages/Clojure` directory?  If so, you might want to check out [this way](https://github.com/vigetlabs/otp/blob/d29d87978b24ef67f1d70dd6d0072d4314616c25/languages/Julia/decrypt#L4) of getting the path to the current script.